js/impact.js

```diff
@@ -738,7 +738,7 @@ var GLPIImpact = {
             {
                 selector: 'node:parent',
                 style: {
-                    'padding'           : '30px',
+                    'padding'           : '50px',
```

Comment:
Could you show us why this padding changes is needed with a before/after screenshot ?
I can't test it directly as I encountered some errors.

Reply:
![image](https://github.com/glpi-project/glpi/assets/13178158/7b3834a7-0783-426f-b302-e86e35c28093)